packages/react-ui-ag/src/Gmap/Markers.js

```diff
@@ -52,7 +54,7 @@ export default class Markers extends PureComponent {
           if (onMouseOut) onMouseOut(marker)
         },
         ...this.props.marker(feature),
-        zIndex: !isBasic ? 2 : 1,
```

Comment:
Is the `isBasic` logic no longer needed?

Reply:
NM, I see that's the whole point of the PR!

Reply:
haha `isBasic` is still used to determine whether its a red or a grey icon